weather_app/admin.py

```diff
@@ -0,0 +1,27 @@
+from django.contrib import admin
+from .models import WeatherCharacteristic,Weather
+
+
+class WeatherCharacteristicAdminClass(admin.ModelAdmin):
+
+    list_display = [
+        'max_value',
+        'min_value',
+        'avg_value',
+    ]
+
+    list_filter = [
```

Comment:
no need to add such a huge value column for filtering its baseless, try adding a custom filter for ranges of temperature